### DISCUSSION TITLE
standard single indicator through-out the MRF

### DISCUSSION THREAD

**Discussion by MS2606** [CMSgov/hospital-price-transparency#100] [Q&A] [answered]
There is a mention of using single indicator in the FAQs through-out the MRF for the cells where the charges or rates are unavailable. for example gross charges for surgery or MS DRGs where CMS does not want hospitals to average the charges. Could you expand on what would be the acceptable single indicator for these blank cells so that MRF can run through validator without having any missing property errors. Thank you

**Comment by RidenourK** [comment, marked as answer]
@MS2606. Thank you for your question. 

If a hospital has not established **any** of the five standard charges for a hospital item or service, the hospital is not required to include that item or service in the MRF. Please only include hospital items and services for which you have established a standard charge. 

As indicated on this Hospital Price Transparency – Data Dictionary GitHub repository, the following are additional reminders to avoid common errors in MRFs:

- Encode valid values as instructed in the Hospital Price Transparency- Data Dictionary GitHub repository. Values encoded incorrectly will generate a deficiency.
- Do not insert a value or any type of indicators (e.g., “N/A” or “0”) if the hospital does not have applicable data to encode. If you would like to include an explanation for the blanks, you may do so using Additional Generic Notes or Additional Payer-Specific Notes.
- If the valid value is ‘numeric’ (such as for Payer-Specific Negotiated Charge: Dollar Amount), inserting anything other than a number (such as inserting a dollar sign with a number) will generate a deficiency. Similarly, if the valid value is ‘enum’ (such as for Code Type), inserting anything other than the values indicated (such as inserting ‘other’) will generate a deficiency.
- All "Numeric" data elements must be positive numbers. Entering a negative number or "0" will generate a deficiency.

Please review the [conditional requirements](https://github.com/CMSgov/hospital-price-transparency/tree/master/documentation/CSV#conditional-requirements) as well.

Updated Hospital Price Transparency FAQs are forthcoming.
